README.md

```diff
@@ -137,10 +138,11 @@ In cases where the project is not able to process JS assets, there are pre-proce
 
 <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../docs/partials/useCases.md) -->
 <!-- The below content is automatically added from ./../docs/partials/useCases.md -->
-Use the `<auro-card>` element to:
+The `<auro-card>` element should be used in situations where users may:
 
-* Grab the attention of a user
-* Direct your user to another resource for further information
+* ...
```

Comment:
Use cases need to be updated

Reply:
See https://github.com/AlaskaAirlines/auro-card/pull/56/files#diff-fbf9bf670db9967a32915ed264a8b2637370e3d768ee7cb126124266ce965626